docs/testing.md

```diff
@@ -1,99 +0,0 @@
-# Testing Foreman Operations Collection
```

Comment:
any reason to delete this file?

(I don't have any hard feelings either way, mostly curious)

Reply:
It looked at me funny.